basic-auth/routes/authorRouter.js

```diff
@@ -31,7 +31,7 @@ router.post("/signup", (req, res, next) => {
       return newUser.save();
     })
     .then(user => {
-      res.redirect("/");
+      res.render("/");
```

Comment:
Esto os da error. Si queréis renderizar index tendréis que poner `res.render("index")`
si lo que queréis es redireccionar sería `res.redirect("/");` no os confundáis.
Cuando utilizáis render tendréis que especificar el nombre del archivo que queréis renderizar, en cambio si utilizáis redirect tendréis que especificar el destino que quieres que tenga.